ui/src/shared/components/cells/Cells.tsx

```diff
@@ -65,16 +80,24 @@ class Cells extends Component<Props & WithRouterProps> {
   }
 
   private get cells(): Layout[] {
+    const {state} = this.props
     return this.props.cells
       .filter(c => c.status === RemoteDataState.Done)
-      .map(c => ({
-        ...c,
-        x: c.x,
-        y: c.y,
-        h: c.h,
-        w: c.w,
-        i: c.id,
-      }))
+      .map(c => {
+        const view = getByID<View>(state, ResourceType.Views, c.id)
```

Comment:
great job. Exactly what I thought, surprised there wasn't a better way but it makes sense

Reply:
sometimes the best path is the easiest